.gitignore

```diff
@@ -7,6 +7,7 @@ coverage.xml
 htmlcov
 reports/*
 .tox/*
+/dist
```

Comment:
Why the leading slash? I don't know that I've seen that before.

Reply:
This only ignores `dist` if it's in the same directory as the gitignore file (in this case, the repo root).

Reply:
Looks like the cookiecutter does the same thing: https://github.com/openedx/edx-cookiecutters/blob/master/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D/.gitignore#L11